packages/api-database/source/models/transaction.ts

```diff
@@ -103,7 +103,7 @@ export class Transaction {
 		type: "jsonb",
 		// TODO: separate tables for 1:n assets
 	})
-	public asset!: Record<string, any> | undefined;
+	public asset: Record<string, any> | undefined | null;
```

Comment:
Can this be modified in a way to only return single type undefined or null in addition to the string?